machine/machine.go

```diff
@@ -51,24 +61,68 @@ func (r *Machine) Ping() error {
 	return nil
 }
 
+// StopAgent implements ControllerNode.StopAgent.
+func (r *Machine) StopAgent() error {
```

Comment:
StopAgent and StartAgent are the same other than the command (and the error logging) - pull out a helper?